ProjectBank/Client/Shared/NavMenu.razor

```diff
@@ -1,50 +1,56 @@
         <a class="navbar-brand" href="">ProjectBank</a>
         <button title="Navigation menu" class="navbar-toggler" @onclick="ToggleNavMenu">
             <span class="navbar-toggler-icon"></span>
-        </button>
+        </button> 
     </div>
-</div>
-
-<div class="@NavMenuCssClass" @onclick="ToggleNavMenu">
-    <nav class="flex-column">
-        <div class="nav-item px-3">
-            <NavLink class="nav-link" href="" Match="NavLinkMatch.All">
-                <span class="oi oi-home" aria-hidden="true"></span> Home
-            </NavLink>
-        </div>
-        <div class="nav-item px-3">
-            <NavLink class="nav-link" href="counter">
-                <span class="oi oi-plus" aria-hidden="true"></span> Counter
-            </NavLink>
-        </div>
-        <div class="nav-item px-3">
-            <NavLink class="nav-link" href="fetchdata">
-                <span class="oi oi-list-rich" aria-hidden="true"></span> Fetch data
-            </NavLink>
-        </div>
-        <div class="nav-item px-3">
-            <NavLink class="nav-link" href="project/1/">
-                <span class="oi oi-list-rich" aria-hidden="true"></span> Projects
-            </NavLink>
-        </div>
-        <div class="nav-item px-3">
-            <NavLink class="nav-link" href="MyProjects">
-                <span class="oi oi-list-rich" aria-hidden="true"></span> My Projects
-            </NavLink>
-        </div>
-    </nav>
-</div>
+   
+    <div class="@NavMenuCssClass">
+        <div class="@NavMenuFlexCssClass "> 
+            <div class="nav-item p-3">
+                <NavLink class="nav-link" href="" Match="NavLinkMatch.All">
+                    <span class="oi oi-home" aria-hidden="true"></span> Explore
+                </NavLink>
+            </div>
+            <div class="nav-item p-3">
+                <NavLink class="nav-link" href="counter">
+                    <span class="oi oi-plus" aria-hidden="true"></span> Create Project
+                </NavLink>
+            </div>
+            <div class="nav-item p-3">
+                <NavLink class="nav-link" href="project/1/">
+                    <span class="oi oi-list-rich" aria-hidden="true"></span> Projects
+                </NavLink>
+            </div>
+            <div class="nav-item p-3" @onclick="SwitchLogIn">
+                <NavLink class="nav-link" href="Login">
+                    <span class="oi oi-account-@User" aria-hidden="true"></span> @User
+                </NavLink>
+            </div>
+        </div> 
+    </div>   
+
+</nav>
 
 @code {
     private bool collapseNavMenu = true;
 
-    private string? NavMenuCssClass => collapseNavMenu ? "collapse" : null;
+    private bool loggedin = true;
+
+    private string? NavMenuCssClass => collapseNavMenu ? "collapse col-md-auto" : null;
+    private string? NavMenuFlexCssClass => collapseNavMenu ? "d-flex justify-content-end" : null;
+    private string? NavMenuColCssClass => collapseNavMenu ? "col" : null;
```

Comment:
I don't know where it happened but is it possible we could get the hamburger menu to fold out from either side (preferably right) instead of folding down from the top? Like the horizontal toggle here: https://getbootstrap.com/docs/5.1/components/collapse/

Reply:
@frejabv im not sure which toggle you refer to. 
The blue button that says "toggle with collapse"? It is possible - but then we will have a permanent blue box where the menu is hidden.

Reply:
Alright then it doesn't matter!